### PR TITLE
fix: bolt only support hessian2

### PR DESCRIPTION
**Pull request by gxcsoccer**
bolt 协议里面只有 hessian2 没有 hessian 了，而且 hessian2 的 code 是 1

---
This change is [Reviewable](https://reviewable.io/reviews/alipay/sofa-bolt-node/4)

### CODE REVIEW

File: lib/codec/hessian/app_type/rpc_server_exception.js [hunk 1]
```diff
@@ -1,12 +1,9 @@
 'use strict';
 
-const compile = require('../compile');
+const compile = require('sofa-hessian-node').compile;
```
Comment (gxcsoccer) [inline lib/codec/hessian/app_type/rpc_server_exception.js]: 用 sofa-hessian-node 替换

File: lib/protocol/const.js [hunk 2]
```diff
@@ -4,14 +4,12 @@ const ResponseStatus = require('./enum/response_status');
 
 // 序列化相关
 exports.codecName2Code = {
-  hessian: 1,
-  hessian2: 4,
+  hessian2: 1,
   protobuf: 11,
```
Comment (gxcsoccer) [inline lib/protocol/const.js]: 最关键就是这里 hessian2 用的 1

**Comment by gxcsoccer** [comment]
这算是个bug，先合并fix 了

**Comment by gxcsoccer** [comment]
1.0.2

**Comment by kevinfuture** [comment]
赞赞

**Pull request merged**